package/MDAnalysis/coordinates/H5MD.py

```diff
@@ -673,10 +675,13 @@ def _read_frame(self, frame):
     def _copy_to_data(self):
         """assigns values to keys in data dictionary"""
 
-        if 'observables' in self._file:
-            for key in self._file['observables'].keys():
-                self.ts.data[key] = self._file['observables'][key][
-                    'value'][self._frame]
+        if "observables" in self._file:
```

Comment:
We should issue a warning that observables of the form `observables/group1/observable1` are being ignored if present.

Reply:
Add this limitation to the documentation.

Reply:
We should be reading all "valid" observables with the current implementation.